suite/spdz/src/main/java/dk/alexandra/fresco/suite/spdz/datatypes/SpdzSInt.java

```diff
@@ -2,23 +2,130 @@
 
 import dk.alexandra.fresco.framework.value.SInt;
 import java.io.Serializable;
+import java.math.BigInteger;
 
 public class SpdzSInt implements SInt, Serializable {
 
-  private static final long serialVersionUID = -9048612329603301195L;
-  public SpdzElement value;
+  private static final long serialVersionUID = 8828769687281856043L;
```

Comment:
Do we actually use this as a serializable? we may not need to implement that interface.